samples/snippets/src/main/java/com/example/logging/ListLogs.java

```diff
@@ -22,10 +22,10 @@
 import com.google.cloud.logging.Logging.EntryListOption;
 import com.google.cloud.logging.LoggingOptions;
 
-/** List logs programmatically using the StackDriver Logging API. */
+/** List logs programmatically using the Cloud Logging API. */
 public class ListLogs {
 
-  /** Expects an existing Stackdriver log name as an argument. */
+  /** Expects an existing Cloud log name as an argument. */
   public static void main(String... args) throws Exception {
     // [START logging_list_log_entries]
     // Instantiates a client
```

Comment:
maybe a comment on line 37 to suggest filtering by timestamp/other indeces really speeds up the query?

Reply:
I mentioned this above, but do you think we should add the timestamp by default instead of just a comment?

Reply:
Good point. I'll include it. I didn't intend to change the samples here - we'll need to rework them in the future, but the comment will be useful and will add.